library-best-practices/JLBP-13.md

```diff
@@ -0,0 +1,25 @@
+[JLBP-13] Quickly remove references to deprecated features in dependencies
+--------------------------------------------------------------------------
+
+- This best practice refers to code that *uses* deprecated functionality from
+  another library, not to the deprecated functionality itself.
+  - Consequently, this best practice does not apply to libraries that have no
+    dependencies, and does not apply to usage of deprecated features within
+    a library.
+  - A "usage" in this context is any reference to a class or method in
```

Comment:
I'm not sure we need to define this here. It seems self-evident in context.

Reply:
Ok - I may have overcompensated based on your prior comment. I'll remove it.